docs/apis-tools/testing/assertions.md

```diff
@@ -241,3 +245,43 @@ Assert that the process instance has the given variables. The assertion fails if
 Map<String, Object> expectedVariables = //
 assertThat(processInstance).hasVariables(expectedVariables);
 ```
+
+## Custom assertions
+
+You can build your own assertions similar to the assertions from CPT.
+
+- Use the preconfigured Camunda client to retrieve the process data.
```

Comment:
Can we link to the Camunda client you are referring to here? The Java client?

Reply:
Yes, the Camunda Java client. The usage of the client is introduced on the [Getting started](https://preview.docs.camunda.cloud/pr-5519/docs/next/apis-tools/testing/getting-started/#write-a-test) page. I see no need to link to the client page.

Reply:
Would it be better to link to this [getting started](https://preview.docs.camunda.cloud/pr-5519/docs/next/apis-tools/testing/getting-started/#write-a-test) page from this page then? If a user opens this page or is linked to this page but hasn't yet visited the getting started page, they may wonder where to go for the preconfigured client.

Reply:
I assume that users would read the getting started page first to get an understanding of the testing library before looking at the assertions.